.travis.yml

```diff
@@ -6,20 +6,12 @@ cache:
 
 env:
   global:
-    - COMPOSER_ARGS="--no-interaction"
+    - COMPOSER_ARGS="--no-interaction --ignore-platform-reqs"
     - COVERAGE_DEPS="php-coveralls/php-coveralls"
 
 matrix:
   fast_finish: true
   include:
-    - php: 7.2
-      env:
-        - DEPS=lowest
-    - php: 7.2
-      env:
-        - DEPS=latest
```

Comment:
These flags need to be ported over to 7.3

Reply:
which ones? cs check and test_coverage?

Reply:
i have added them back in :-)